x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -190,7 +191,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
     };
   }
 
-  public start(core: CoreStart, plugins: StartPlugins) {
+  public async start(core: CoreStart, plugins: StartPlugins) {
```

Comment:
We need to avoid asynchronous methods for the plugin start and setup according to the guidelines.